README.md

```diff
@@ -48,3 +48,34 @@ Required or forbidden package references are configured using item groups which
 ## Security
 
 Please see our [security policy](https://github.com/G-Research/NuGetPackageChecker/blob/main/SECURITY.md) for details on reporting security vulnerabilities.
+
+## Contributing
```

Comment:
Instead of duplicating contribution and issue reporting info in both the README and CONTRIBUTING files, link to the CONTRIBUTING guide.  


We welcome your contributions and feedback. For more information about contributing to this project see the [Contributing Guide](CONTRIBUTING.md) and review the [Code of Conduct](CODE_OF_CONDUCT.md).